### ISSUE TITLE
[BUG] - *Multiplayer Reward*

### ISSUE THREAD

**Issue by MynameisCrii** [Kikiisyourfriend/ProjectIsothermalExpert#57]
### **1.16.5:**

```
1.0.8  (Latest)
```
### **Are you on a server? Or in Singleplayer?:**

```
LAN Server
```

### **Description:**

```
I was playing this modded pack with some friends of mine until we find ourselves having to take the rewards with coins. When we go to exchange the type of coin (Example: Exchange the Chaos Orb for the Exalted Orb) it gives the changed coin to everyone thus making it possible to run a bug to have infinite coins. Same thing happens with items taken with money, if a player takes a reward he will give it automatically to everyone. (Sorry for the grammar, I'm using the translator)
```

### **Steps to reproduce:**

```
1 Play with another player
2 Get enough coins to exchange them for another type of currency
2 Buying an item with money
```
### **Crash log (*if any*):**
> *Must use a code-pasting website such as Pastebin, or a Gist.*

```
I don't have any crash
```

**Comment by MynameisCrii** [comment]
> ### **1.16.5:**
> ```
> 1.0.8  (Latest)
> ```
> 
> ### **Are you on a server? Or in Singleplayer?:**
> ```
> LAN Server
> ```
> 
> ### **Description:**
> ```
> I was playing this modded pack with some friends of mine until we find ourselves having to take the rewards with coins. When we go to exchange the type of coin (Example: Exchange the Chaos Orb for the Exalted Orb) it gives the changed coin to everyone thus making it possible to run a bug to have infinite coins. Same thing happens with items taken with money, if a player takes a reward he will give it automatically to everyone. (Sorry for the grammar, I'm using the translator)
> ```
> 
> ### **Steps to reproduce:**
> ```
> 1 Play with another player
> 2 Get enough coins to exchange them for another type of currency
> 2 Buying an item with money
> ```
> 
> ### **Crash log (_if any_):**
> > _Must use a code-pasting website such as Pastebin, or a Gist._
> 
> ```
> I don't have any crash
> ```

**Comment by Kikiisyourfriend** [comment]
Yeah... I think I tried looking for a fix and never found one so enjoy all the phat loot as a group!
